### PR TITLE
update to SOCProcessFramework solution

### PR DESCRIPTION
**Pull request by BenjiSec**
Required items, please complete
   
   Change(s):
   - Updated Get-SOCActions playbook and SocRA watchlist to support Microsoft Sentinel incident tasks
   - Update to ReadMe file
   -
   - Incident trigger for Get-SentinelAlertsEvidence playbook
   - Update to ReadMe file

   Reason for Change(s):
   - Support new features in Sentinel

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @BenjiSec looking into this

**Comment by v-prasadboke** [comment]
Hello @rahul0216 and @manishkumar1991 can you please review the playbook and approve

File: Solutions/SOC-Process-Framework/Playbooks/readme.md [hunk 1]
```diff
@@ -22,4 +22,4 @@ After deploying the playbook, you must authorize the connections leveraged.
 2. Under "Development Tools" (located on the left), click "API Connections".
 3. Ensure each connection has been authorized.
```
Comment (manishkumar1991) [inline Solutions/SOC-Process-Framework/Playbooks/readme.md:24]: We think that for playbook to add task to incident , it should have "sentinel responder role" assigned .

Kindly add the steps here on how to assign the sentinel responder role .

Ref : https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-Enrichment/readme.md

**Comment by v-prasadboke** [comment]
Hello @BenjiSec manishkumar has requested some changes

File: Solutions/SOC-Process-Framework/Playbooks/readme.md (hunk 1 above)
Comment (BenjiSec) [reply Solutions/SOC-Process-Framework/Playbooks/readme.md:25]: Hi, done

**Comment by BenjiSec** [comment]
> Hello @BenjiSec manishkumar has requested some changes

done

**Comment by v-prasadboke** [comment]
Hello @manishkumar1991 waiting for your feedback

**Review by manishkumar1991 — APPROVED** [review]
approved

**Comment by v-prasadboke** [comment]
Thank you @manishkumar1991,
    Continuing to review this pr

**Comment by v-prasadboke** [comment]
Hello @BenjiSec still working on this

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**